cnd/migrations/2019-11-11-034058_create-message-tables/down.sql

```diff
@@ -1,10 +0,0 @@
--- This file should undo anything in `up.sql`
-
-DROP TABLE rfc003_bitcoin_ethereum_bitcoin_ether_request_messages;
```

Comment:
You can't just delete the old migrations unfortunately :/

But you can add one that drops the tables :)

Reply:
I think that the next version of cnd should explicitly _not_ provide an upgrade path. I rekon we just clobber everything rfc003 including the database migrations. If folks want their database for rfc003 swaps they should fire up v0.8 to get them. In reality, since we know of no one doing mainnet swaps this is not going to adversely effect anyone.

Reply:
> You can't just delete the old migrations unfortunately :/

Am I missing some reason why you say we cannot do this?